app/views/items/_item_screen.html.erb

```diff
@@ -0,0 +1,188 @@
+<div class="container mb-5">
```

Comment:
This div is closed outside of this document (in show.html.erb). Possible solutions:
- Put the `<div />` in this file.
- Put all the HTML in `show.html.erb `directly. I would prefer this because the file `_item_screen.html.erb` is only used in one place and the separation doesn't add any value in my opinion.